data-sets-api-server/src/main/java/org/zowe/unix/files/services/zosmf/DeleteUnixFileRunner.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright IBM Corporation 2019
+ */
+package org.zowe.unix.files.services.zosmf;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.RequestBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.zowe.api.common.connectors.zosmf.ZosmfConnector;
+import org.zowe.api.common.exceptions.ZoweApiRestException;
+import org.zowe.api.common.utils.ResponseCache;
+import org.zowe.unix.files.exceptions.FileNotFoundException;
+import org.zowe.unix.files.exceptions.NotAEmptyDirectoryException;
+import org.zowe.unix.files.exceptions.PermissionDeniedFileException;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class DeleteUnixFileRunner extends AbstractZosmfUnixFilesRequestRunner<Void> {
+    
+    @Autowired
+    ZosmfConnector zosmfConnector;
+
+    private String path;
+    private boolean isRecursive;
+    
+    public DeleteUnixFileRunner(String path) {
```

Comment:
For constructors the precedent is to leverage Lombok @AllArgsConstructor @NoArgsConstructor

Reply:
using @AllArgsConstructor except in default value case